doc/lua_api.txt

```diff
@@ -7859,6 +7862,16 @@ Used by `minetest.add_particle`.
         glow = 0
         -- Optional, specify particle self-luminescence in darkness.
         -- Values 0-14.
+
+        node = {name = "ignore", param1 = 0, param2 = 0},
+        -- Optional, if specified the particle will have the same appearance as
+        -- node dig particles for the given node.
+        -- `texture` and `animation` will be ignored if this is set.
+
+        node_tile = 0,
+        -- Optional, only valid in combination with `node`
+        -- If set to a valid number 1-6, specifies the tile from which the
+        -- particle texture is picked. Otherwise, the default behavior is used.
```

Comment:
This last sentence sounds ambiguous to me - what's the default behaviour here?

Reply:
The default behaviour is to pick a random tile.
I intentionally meant to keep this unspecified because default behavior might change in the future and we don't want to guarantee it.

Reply:
Ahh, that's what I thought.

> I intentionally meant to keep this unspecified because default behavior might change in the future and we don't want to guarantee it.

Understood.

Reply:
I agree that the documentation is a bit confusing, maybe you have a suggestion how to word this better?

Reply:
The following would suffice, IMO:

`... Otherwise, a random tile is selected.`

(Btw, is the random selection per-particle or per-particlespawner for particlespawners?)

Maybe also add a note about this behaviour changing in the future?

`Note: This behaviour isn't final, and might change in the future.`

... or just 

`Note: This behaviour might change in the future.`

Reply:
> Btw, is the random selection per-particle or per-particlespawner for particlespawners?

per particle

Reply:
I see, thanks